lib/si-events-rs/src/func_run.rs

```diff
@@ -66,6 +66,7 @@ pub enum FuncBackendKind {
     JsAttribute,
     JsAuthentication,
     Json,
+    // NOTE(nick): this has been deprecated. Not adding serde deprecated tag in case it affects the type.
     JsReconciliation,
```

Comment:
It looks like these are postcard serialized as well: https://github.com//systeminit/si/blob/nick/eng-2653/lib/si-layer-cache/src/db/func_run_log.rs#L119

So let's remove the remain sorted here and also add a note about postcard footguns

Reply:
Gotcha. Will take the same action as the above.